lang/nl/navbar.php

```diff
@@ -5,6 +5,7 @@
     'manage_accounts' => 'Beheer accounts',
     'manage_products' => 'Beheer producten',
     'manage_orders' => 'Beheer bestellingen',
+    'manage_groups' => 'Groepen',
```

Comment:
```suggestion
    'manage_groups' => 'Beheer groepen',
```

Reply:
Ik had dit eerst ook, maar je kan nog niks beheren in deze US

Reply:
Hmmm maar hij staat wel onder beheren...
Kan wel zo, alleen dan moet de dropdown wel terug naar 'Admin'